### PR TITLE
PGPT-91: Apply new design to the main page. Part 2.

### PR DESCRIPTION
**Pull request by AlexandrOgarkov**

### CODE REVIEW

File: apps/chat/src/components/Chat/ModelsDialog.tsx [hunk 1]
```diff
@@ -175,72 +174,44 @@ export const ModelsDialog: FC<ModelsDialogProps> = ({
...
-          {`${t('Applications')} (${typeCounts.application})`}
-        </button>
+        <ContextMenu
+          menuItems={menuItems}
+          triggerIconClassName="absolute right-8 md:right-12 cursor-pointer max-h-[38px]"
+          TriggerCustomRenderer={
+            <Tooltip tooltip={t('Search filter')} hideTooltip={isOpen}>
+              <div className="flex items-end text-quaternary-bg-light hover:text-primary-bg-light">
+                <FilterIcon width={20} height={20} className="inline-block" />
+                <span className="hidden pl-2 md:inline-block">Filters</span>
```
Comment (viktoriia-mindyk-pr) [inline apps/chat/src/components/Chat/ModelsDialog.tsx:210]: add translation for "Filters"

File: apps/chat/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -119,29 +119,15 @@ const Header = () => {
...
             className="mr-3 self-center md:ml-5"
             width={31}
             height={30}
           />
           <div className="my-3 border-r border-primary" />
           <SecondaryLogo className="ml-3 self-center" width={25} height={30} />
-          <span className="ml-2 flex flex-wrap content-center text-[22px] font-extrabold md:mr-3">
-            prGPT
+          <span className="ml-2 flex flex-wrap content-center font-weave text-[22px] font-bold md:mr-3">
+            PR GPT
```
Comment (viktoriia-mindyk-pr) [inline apps/chat/src/components/Header/Header.tsx:130]: create constant for it, cause it's used 2 times in the code

File: apps/chat/src/components/Chat/ModelsDialog.tsx [hunk 3]
```diff
@@ -165,6 +126,44 @@ export const ModelsDialog: FC<ModelsDialogProps> = ({
...
   );
 
+  const onClickMenuItemHandler = (entityType: any) => {
+    handleFilterType(entityType);
+  };
+
+  const menuItems: DisplayMenuItemProps[] = [
+    {
+      display: true,
+      name: 'Models',
```
Comment (viktoriia-mindyk-pr) [inline apps/chat/src/components/Chat/ModelsDialog.tsx:136]: check only if in the ContextMenu we have translations for name

**Review by viktoriia-mindyk-pr — APPROVED** [review]

File: apps/chat/src/components/Chat/ModelsDialog.tsx (hunk 1 above)
Comment (AlexandrOgarkov) [reply apps/chat/src/components/Chat/ModelsDialog.tsx]: Thank you, updated

File: apps/chat/src/components/Chat/ModelsDialog.tsx (hunk 3 above)
Comment (AlexandrOgarkov) [reply apps/chat/src/components/Chat/ModelsDialog.tsx]: No, we pass this name as it is. Added translation. Thank you

**Pull request merged**